jupyterlab-server-proxy/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
On purpose or a accidental change?

Reply:
Om purpose, the reason is described in the second commit's commit messages